src/components/common/ui/form/numbered-text-area-mui/components/LineNumbers.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+export const lineNumbers: SxProps<Theme> = {
+  marginTop: '9px',
+  textAlign: 'center',
+  width: '30px',
+  display: 'flex',
+  flexDirection: 'column',
+  typography: 'body1',
+  lineHeight: '24px',
```

Comment:
You already used `typography`, why did you wrote lineHeight ?

Reply:
Because as a line number, the height needs to match `body2` (text area) line height.